server/router/index.js

```diff
@@ -3,25 +3,24 @@ const express = require('express');
 const router = express.Router();
 const taskController = require('../controller/taskController');
 const userController = require('../controller/userController');
-const authorizationJWT = require('../middlewares/auth');
 //TODO: remove
 router.get('/test', userController.test);
 
 
-router.get('/api/tasks', authorizationJWT, taskController.getTasks);
+router.get('/api/tasks',  taskController.getTasks);
 
-router.get('/api/task/:id',authorizationJWT, taskController.getTask);
+router.get('/api/task/:id', taskController.getTask);
 
-router.post('/api/task',authorizationJWT, taskController.addTask);
+router.post('/api/task', taskController.addTask);
 
-router.post('/api/task/:id',authorizationJWT,taskController.updateTask);
+router.post('/api/task/:id', taskController.updateTask);
 
-router.delete('/api/task/:id',authorizationJWT, taskController.deleteTask);
+router.delete('/api/task/:id', taskController.deleteTask);
 
-router.post('/api/register', userController.register);
```

Comment:
why ?

Reply:
These semicolons are hard ... but added this one back